README.md

```diff
@@ -7,7 +7,7 @@ Invite is a Django app for inviting new users to your new or existing Django pro
 ## Dependencies
 
 * Python 2.7.x
-* Django 1.5 - 1.7
+* Django 1.8 (Please use django-invite 1.0.2 for Django 1.5-1.7)
 
 ## Documentation
 
```

Comment:
We could add in steps for 
`./manage.py migrate`
`./manage.py createsuperuser`
before the  "Run the development server." step.